src/main/java/uk/gov/hmcts/reform/iacaseaccessapi/domain/services/SupplementaryDetailsService.java

```diff
@@ -5,5 +5,5 @@
 
 public interface SupplementaryDetailsService {
 
-    List<SupplementaryInfo> getSupplementaryDetails(List<String>  ccdCaseNumberList);
+    List<SupplementaryInfo> getSupplementaryDetails(List<String>  ccdCaseNumberList,String authorisation);
```

Comment:
we don't need to pass authorisation to the service

Reply:
Parameter removed